ansible/macos/macos11.yml

```diff
@@ -0,0 +1,185 @@
+
+      - name: Configure docker-machine-nfs
+        command: /usr/local/bin/docker-machine-nfs dinghy
+
+      - name: Create resolver directory
+        file: path=/etc/resolver state=directory mode=0755
+        become: yes
+        become_method: sudo
+
+      - name: Delete docker.loc old file if present
+        file: path=/etc/resolver/docker.loc state=absent mode=0755
+        become: yes
+        become_method: sudo
+
+      - name: Create loc resolver file
+        file: path=/etc/resolver/loc state=touch mode=0755
+        become: yes
+        become_method: sudo
+
+      - name: Add configuration to /etc/resolver/loc
+        lineinfile: dest=/etc/resolver/loc state=present line="nameserver 127.0.0.1"
+        become: yes
+        become_method: sudo
+
+      - name: Start dnsdock container
+        shell: "{{ item }}"
+        with_items:
+          - "eval $(dinghy env) && docker rm -vf dnsdock || true"
+          - "eval $(dinghy env) && docker rm -vf dinghy_http_proxy || true"
+          - "eval $(dinghy env) && docker run --restart=always -d -v /var/run/docker.sock:/var/run/docker.sock --name dnsdock -p 172.17.0.1:53:53/udp {{ dnsdock_image }}"
+
+      - name: Add static routing
+        shell: "route delete 172.17.0.0 && route delete 172.16/12 && route -n add -net 172.16.0.0/12 {{ ip_machine.stdout }}"
+        become: yes
+        become_method: sudo
+        register: command_result
+        failed_when: "'FAILED' in command_result.stderr"
+
+      - name: Clear dns cache
+        command: killall -HUP mDNSResponder
+        become: yes
+        become_method: sudo
+
+      - name: Add staff group to sudoers to run route command with no password
+        lineinfile:
+          dest: /etc/sudoers
+          state: present
+          line: '%staff ALL=(ALL) NOPASSWD: /sbin/route'
+        become: yes
+        become_method: sudo
+
+      - name: Enable IP FORWARD
+        shell: 'docker-machine ssh dinghy -t sudo ''sudo chmod uo+w /var/lib/boot2docker/bootlocal.sh;  echo "sysctl net.ipv4.ip_forward=1" >> /var/lib/boot2docker/bootlocal.sh'''
+
+      - name: Set static iptables to FORWARD ACCEPT
+        shell: "docker-machine ssh dinghy sudo iptables -P FORWARD ACCEPT"
+
+      - name: Create launchctl script to run docker-machine and add routing tables
+        file:
+          src: "{{ dev_env_dir }}/config/macosx/com.docker.machine.dev.plist"
+          dest: ~/Library/LaunchAgents/com.docker.machine.dev.plist
+          state: link
+          force: yes
+
+      - name: Test DNSDOCK - Run dummy container
+        shell: eval "$(dinghy env)" && docker run -d -e DNSDOCK_ALIAS=test1.redis.docker.loc --name redis-test redis:alpine
+
+      - name: Test DNSDOCK - Run dummy container
+        shell: ping -c1 test1.redis.docker.loc
+
+      - name: Test DNSDOCK - Run dummy container
+        shell: eval "$(dinghy env)" && docker kill redis-test && docker rm redis-test
```

Comment:
This file for big sur for now it should empty or just copy the the binary, like this one: https://github.com/sparkfabrik/sparkdock/blob/master/ansible/ubuntu/ubuntu20.yml#L70